### ISSUE TITLE
Is it possible to load columnfamily on demand when starting rocksdb?

### ISSUE THREAD

**Issue by iokk3732** [facebook/rocksdb#12881]
> Note: Please use Issues only for bug reports. For questions, discussions, feature requests, etc. post to dev group: https://groups.google.com/forum/#!forum/rocksdb or https://www.facebook.com/groups/rocksdb.dev

### Steps to reproduce the behavior

Can I load columnfamily on demand when RocksDB starts? Now I need to use RocksDB.listColumnFamilies() to traverse all columnfamilies every time the program starts, but I have created many columnfamilies (one for each user), which makes my program start very slowly. Can I load on demand and load and open a columnfamily when I really need it?

If I use Rocksdb.open() without loading all of them, I will get: org.rocksdb.RocksDBException: Column families not opened

**Comment by ltamasi** [comment]
No, this is not supported when opening the db in read/write mode; see https://github.com/facebook/rocksdb/wiki/Column-Families:

> When opening a DB in a read-write mode, you need to specify all Column Families that currently exist in a DB. If that's not the case, DB::Open call will return Status::InvalidArgument().
